doc/Doxyfile-common

```diff
@@ -206,7 +206,7 @@ OPTIMIZE_OUTPUT_FOR_C  = YES
 # Java. For instance, namespaces will be presented as packages, qualified
 # scopes will look different, etc.
 
-OPTIMIZE_OUTPUT_JAVA   = NO
+OPTIMIZE_OUTPUT_JAVA   = YES
```

Comment:
Within the documentation for Doxygen, it is suggested to set `OPTIMIZE_OUTPUT_JAVA` to `YES` when documenting Python code. Please see the following [link](https://www.doxygen.nl/manual/docblocks.html#pythonblocks).

Reply:
This was removed due to not being needed.